nova/files/mitaka/nova-compute.conf.Debian

```diff
@@ -186,3 +186,5 @@ timeout=30
 os_region_name = {{ compute.identity.region }}
 catalog_info=volumev2:cinderv2:internalURL
 
+[workarounds]
```

Comment:
I would better wrap this section into `if compute.workarounds is defined` conditional to avoid having empty section